docs/conf.py

```diff
@@ -88,7 +92,11 @@
 extensions.append('nbsphinx')
 extensions.append('IPython.sphinxext.ipython_console_highlighting')
 extensions.append('sphinx.ext.mathjax')
-nbsphinx_execute = 'never'
+if eval(setup_cfg.get('execute_notebooks')):
```

Comment:
This looks weird. Why take one boolean `execute_notebooks` and convert it to a two-choice `nbsphinx_execute = {'always', 'never'}`? Suggest to just put what you want in `setup.cfg` and then use it directly.

Also: calling `eval` here really isn't needed. If `setup_cfg.get` doesn't yield a bool directly, then pass the string through the `bool` function:
```
execute_notebooks = bool(setup_cfg.get('execute_notebooks'))
```